src/weights.jl

```diff
@@ -193,6 +193,60 @@ pweights(vs::RealArray) = ProbabilityWeights(vec(vs))
     end
 end
 
+"""
+    eweights(t::AbstractVector{<:Integer}, λ::Real)
```

Comment:
Why restrict this to integers, while the next method accepts any `T`? Shouldn't `Real` be used in both cases?

Reply:
In this method each element of `t` represents a index in time. In the method below we use `T` because we're generating the weights from the indices of `t` in `r`.

Reply:
OK. Shouldn't the docstring mention `r` somewhere?